big_tests/test.config

```diff
@@ -18,6 +18,7 @@
 %%       so that we rein the "bag of things" approach
 {hosts, [{mim,  [{node, mongooseim@localhost},
                  {domain, <<"localhost">>},
+                 {host_type, <<"localhost">>},
```

Comment:
This isn't necessary, see below.

Reply:
I would rather have it defined like that, not as rpc-s.

Reply:
For me it's better to have one helper that consistently reads it from `mongoose_domain_core` instead of defining it multiple times for multiple hosts. Anyway, as the helper is already there, let's use it consistently. This way we can change the implementation later when needed.